### DISCUSSION TITLE
Right-click context menu for Layers

### DISCUSSION THREAD

**Discussion by jimkring** [ni/labview-icon-editor#187] [General]
As a LabVIEW User, I want to be able to take action on the selected Layers via the right-click context menu.

Currently, I have to select them, and then go to the Layers menu from the top menubar.  It often takes me a little while to remember where the menu is located and that this is even a possibility.

By putting the menu items in a right-click context menu, it will allow me to more easily manage my layers.

![image](https://github.com/ni/labview-icon-editor/assets/381432/9c4facf4-6aac-49ea-87e9-968e4adf4c05)

**Comment by JayKayAce** [comment]
Reusing the menu makes more sense if we remove the first option "Show Layers Page". 
Is there any reason for not just always showing the layers in the editor, instead of having a hidden page, that you need to enable?

**Comment by RobustoSystems** [comment]
I support always showing the Layers page. This would enable the removal of the "Show Layers Page" option.

**Comment by crossrulz** [comment]
Would be adding keyboard shortcuts meet the purpose of this issue? The idea being that we could easily add two more keyboard shortcuts for the Merge Selected Layers and Merge All Layers.

Or is the main issue discoverability of the options?

**Comment by svelderrainruiz** [comment]
Will move to a discussion after talking about it on todays steerco meeting

**Comment by svelderrainruiz** [comment]
@j-medland comments:
 
Removed the "need more clarification" label, there is enough information on this discussion to discuss with LabVIEW r&d
